SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/GridBasedLBStrategy.cpp

```diff
@@ -100,6 +101,37 @@ VirtualWorkerId UGridBasedLBStrategy::WhoShouldHaveAuthority(const AActor& Actor
 	return SpatialConstants::INVALID_VIRTUAL_WORKER_ID;
 }
 
+SpatialGDK::QueryConstraint UGridBasedLBStrategy::GetWorkerInterestQueryConstraint() const
+{
+	// For a grid-based strategy, the interest area is the cell that the worker is authoritative over plus some border region.
+	check(IsReady());
+	check(InterestBorder >= 0);
+	check(Rows * Cols > 0);
+
+	// If there is only one worker, this special cases to a "true" constraint (Component constraint for the position component).
+	if (Rows * Cols == 1)
+	{
+		SpatialGDK::QueryConstraint TrueConstraint;
```

Comment:
I understand that this makes it simpler, but there's a concern that this would lead to functional differences also. We have a WorkersCells entry for the single server, I think we should just use it.

Reply:
sure, removed the special case

Reply:
also removed corresponding test